analyzer/consensus/genesis.go

```diff
@@ -75,12 +70,12 @@ VALUES
 			query += ",\n"
 		}
 	}
-	query += ";"
+	query += `
+ON CONFLICT (id) DO UPDATE SET address = EXCLUDED.address;`
 	queries = append(queries, query)
 
 	// Populate nodes.
-	queries = append(queries, `TRUNCATE {{ChainId}}.nodes CASCADE;`)
-	query = `INSERT INTO {{ChainId}}.nodes (id, entity_id, expiration, tls_pubkey, tls_next_pubkey, p2p_pubkey, consensus_pubkey, roles)
+	query = `INSERT INTO chain.nodes (id, entity_id, expiration, tls_pubkey, tls_next_pubkey, p2p_pubkey, consensus_pubkey, roles)
```

Comment:
Got it, updated to truncate the nodes/allowances as well in addition to debonding_delegations. We don't process committee_member data in the genesis file (understandably, since the network hasn't started yet), so that one was easy :)